ncm-metaconfig/src/main/pan/components/metaconfig/schema.pan

```diff
@@ -15,7 +15,6 @@ type ${project.artifactId}_config =  {
      'mode' : long = 0644
      'owner' : string = 'root'
      'group' : string = 'root'
-     'daemon' ? string[]
```

Comment:
i'm fine with throwing it out of the schema, but a clearer error message is nice (like it is now).
i'm also fine with throwing the functionality out of the code, but there should always be an error logged. ideally, wrt the code, functionality should stay in there for another release (but it can already log warning or error).